cmd/authy-export/authy-export.go

```diff
@@ -149,8 +169,8 @@ func main() {
 		if err != nil {
 			log.Fatalf("Can't write file %v", err)
 		}
-	} else if last4 != ".txt" && last6 != ".wa.txt" {
-		log.Fatalf("Invalid filename %s must end with .html or .txt", filename)
+	} else if last4 != ".txt" && last6 != "raivos" && last6 != ".wa.txt" {
```

Comment:
Isn't the constant literal `".wa.txt"` length seven, such that comparing with the last six will never pass? Why not use `strings.HasSuffix`?

Reply:
@TheChuckster there was a typo, should have been "wa.txt". Fixed. 

> Why not use strings.HasSuffix or filepath.Ext

Feel free to modify the code accordingly.